tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
Currently I'm adding an environment variable to mark when is generating the documentation and skipping calling the configure method. But still being caught by the issue mentioned on the other comment.